src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurBatchUtilityServiceImpl.java

```diff
@@ -0,0 +1,179 @@
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.sql.Date;
+import java.util.Collections;
+
+import javax.xml.bind.JAXBException;
+
+import org.apache.commons.io.IOUtils;
+import org.apache.commons.lang.StringUtils;
+
+import org.kuali.rice.core.api.datetime.DateTimeService;
+import org.kuali.kfs.coreservice.framework.parameter.ParameterService;
+import org.kuali.kfs.gl.GeneralLedgerConstants;
+import org.kuali.kfs.kns.service.DataDictionaryService;
+import org.kuali.kfs.pdp.businessobject.PaymentGroup;
+import org.kuali.kfs.pdp.PdpConstants;
+import org.kuali.kfs.sys.KFSConstants;
+import org.kuali.kfs.sys.batch.BatchInputFileType;
+import org.kuali.kfs.sys.batch.service.BatchInputFileService;
+import org.kuali.kfs.sys.exception.FileStorageException;
+import org.kuali.kfs.sys.service.FileStorageService;
+
+import edu.cornell.kfs.concur.ConcurConstants;
+import edu.cornell.kfs.concur.batch.service.ConcurBatchUtilityService;
+import edu.cornell.kfs.concur.batch.xmlObjects.PdpFeedFileBaseEntry;
+import edu.cornell.kfs.sys.CUKFSConstants;
+import edu.cornell.kfs.sys.CUKFSParameterKeyConstants;
+import edu.cornell.kfs.sys.service.CUMarshalService;
+
+public class ConcurBatchUtilityServiceImpl implements ConcurBatchUtilityService {
+    private static org.apache.log4j.Logger LOG = org.apache.log4j.Logger.getLogger(ConcurBatchUtilityServiceImpl.class);
+    protected BatchInputFileService batchInputFileService;
+    protected DataDictionaryService dataDictionaryService;
+    protected DateTimeService dateTimeService;
+    protected FileStorageService fileStorageService;
+    protected CUMarshalService cuMarshalService;
+    protected ParameterService parameterService;
+
+    public String getConcurParameterValue(String parameterName) {
+        String parameterValue = getParameterService().getParameterValueAsString(CUKFSConstants.ParameterNamespaces.CONCUR, CUKFSParameterKeyConstants.ALL_COMPONENTS, parameterName);
+        return parameterValue;
+    }
+    
+    public void createDoneFile(String fullyQualifiedFileName) throws FileStorageException {
+        getFileStorageService().createDoneFile(fullyQualifiedFileName);
+    }
+    
+    public void removeDoneFile(String fullyQualifiedFileName) throws FileStorageException {
+        getFileStorageService().removeDoneFiles(Collections.singletonList(fullyQualifiedFileName));
+    }
+
+    public String formatPdpPayeeName(String lastName, String firstName, String middleInitial) {
+        String fullName = lastName + KFSConstants.COMMA + KFSConstants.BLANK_SPACE + firstName;
+        if (StringUtils.isNotBlank(middleInitial)) {
+            fullName = fullName + KFSConstants.BLANK_SPACE + middleInitial + KFSConstants.DELIMITER;
+        }
+
+        int payeeNameFieldSize = (getDataDictionaryService().getAttributeMaxLength(PaymentGroup.class, PdpConstants.PaymentDetail.PAYEE_NAME)).intValue();
```

Comment:
would it make sense to just use Integer for this variable and use the int value in the if statement.  The line reads better without the extra parenthesis.